### PR TITLE
Natayap

### PR DESCRIPTION
**Pull request by NatayaP**
May you keep this readme private

### CODE REVIEW

File: lesson_00/.DS_Store
Comment (anthonydmays) [inline lesson_00/.DS_Store:1]: Please remove the .DS_Store files from your PR, here and elsewhere.

File: lesson_00/README.md
Comment (anthonydmays) [inline lesson_00/README.md:1]: Please undo changes to this file, this is not one you should modify.

File: lesson_00/natayaprice/README.md
Comment (anthonydmays) [inline lesson_00/natayaprice/README.md:1]: You need to include a markdown list in your README.

File: lesson_00/natayaprice/README.md [hunk 1]
```diff
@@ -0,0 +1,22 @@
...
+A lover of self-care rituals—indulging in skincare, haircare, and nails. Thriving on the satisfaction of meticulous projects like intricate nail designs, hairstyling, and sewing. Embracing the warmth of summer, reveling in the joy of research, and finding solace in nature. Balancing everything while spending quality time with family and friends. 
+
+## One of My Goals 🌐
+This year, my goal is to travel abroad, and one destination that I have in mind is St. Lucia. Exploring new countries has always intrigued me, and I'm particularly excited about the prospect of visiting the beautiful landscapes and vibrant culture of St. Lucia.
+
+
+## Favorite Book Genres ⭐
+A passionate reader, I explore diverse genres, from science fiction and Afro-futurism to mystery and period pieces. My reading journey is a continuous exploration. Currently immersed in Octavia Butler's "Lilith's Brood."
+
+![My Photo](assets/Photo1.png)
```
Comment (anthonydmays) [inline lesson_00/natayaprice/README.md:19]: The photo link is broken, I think you need to update the path.

**Review by anthonydmays — CHANGES_REQUESTED** [review]
Thanks! Almost there, keep working on it.

**Review by NatayaP — COMMENTED** [review]
updated

File: lesson_00/natayaprice/README.md (hunk 1 above)
Comment (anthonydmays) [reply lesson_00/natayaprice/README.md:24]: Still an issue.

File: lesson_00/README.md
Comment (anthonydmays) [inline lesson_00/README.md:1]: Please restore the original file content.

**Review by anthonydmays — CHANGES_REQUESTED** [review]

**Review by anthonydmays — APPROVED** [review]

**Pull request merged**